tests/usecase/test_jetty_config.py

```diff
@@ -0,0 +1,16 @@
+import basetest
+import json
+
+
+class TestCaseJettyConfig(basetest.BaseTest):
```

Comment:
Can we also add tests for the unhappy paths?
- Invalid JSON in the `JETTY_CONFIG` environment variable
- (maybe) `JETTY_CONFIG` set but empty

Reply:
added extra case with invalid json